database_admin/migrations/035_partition_system_package.up.sql

```diff
@@ -0,0 +1,41 @@
+CREATE OR REPLACE FUNCTION create_table_partitions(tbl regclass, parts INTEGER, rest text)
+    RETURNS VOID AS
+$$
+DECLARE
+    I INTEGER;
+BEGIN
+    I := 0;
+    WHILE I < parts
+        LOOP
+            EXECUTE 'CREATE TABLE ' || text(tbl) || '_' || text(I) || ' PARTITION OF ' || text(tbl) ||
+                    ' FOR VALUES WITH ' || ' ( MODULUS ' || text(parts) || ', REMAINDER ' || text(I) || ')' ||
+                    rest || ';';
+            I = I + 1;
+        END LOOP;
+END;
+$$ LANGUAGE plpgsql;
+
+DROP TABLE IF EXISTS system_package;
+
+CREATE TABLE IF NOT EXISTS system_package
```

Comment:
"IF NOT EXISTS" is useless here.

Reply:
@Josca Hm, this PR is changing faster than I can write review ;).